modules/currency-l0/src/main/scala/org/tessellation/currency/l0/cli/method.scala

```diff
@@ -66,9 +68,32 @@ object method {
 
     val stateAfterJoining: NodeState = NodeState.WaitingForDownload
 
-    val stateChannelAllowanceLists = None
+    val stateChannelAllowanceLists: Option[Map[Address, NonEmptySet[PeerId]]] = environment match {
```

Comment:
This code is duplicated in a few places, can we inherit from common shared path?

Reply:
What part can be extracted? The values used for testnet and integrationnet are different in each case.